### PR TITLE
Test 9.2.4 in CI

### PR DESCRIPTION
**Pull request by jhrcek**

### CODE REVIEW

File: .github/workflows/haskell-ci.yml [hunk 1]
```diff
@@ -108,11 +108,13 @@ jobs:
...
+          node-version: 16
 
       - name: Build and test
         working-directory: frontend
         run: |
           npm install -g create-elm-app@5.22.0
-          elm-app build
-          elm-app test
+          npm elm-app build
+          npm elm-app test
```
Comment (turboMaCk) [inline .github/workflows/haskell-ci.yml:120]: shouldn't this be npx?

Comment (jhrcek) [reply .github/workflows/haskell-ci.yml]: No, I added npm by mistake..

**Comment by jhrcek** [comment]
Ready, including fixing frontend CI.

**Review by michivi — APPROVED** [review]
LGTM

**Pull request merged**